docs/backups.md

```diff
@@ -366,3 +368,56 @@ It is often a good idea to configure retention of older backups with decreasing
 - a monthly backup (retaining 12)
 
 Again, all of these can be assigned to the same backup job. Note that if you do a weekly and a monthly backup, at some point, these will fall on the same day. Xen Orchestra is designed to fail gracefully (with an error message) if a backup job for a VM is already running. For this reason, you will want to set the time on the monthly job to run before the weekly job so that if one fails, it will be the weekly rather than the monthly one; if the weekly one fails, the monthly will be there for that spot in the retention plan; if the monthly one fails, the weekly one will only be retained for 4 weeks, and then there will be a gap in the monthly retention.
+
+## Backup Health Check
+
+Backup health check ensures the backups are ready to be restored.
+
+### Different level of checking
+
+#### Check for boot
+
+XO will restore the VM, either by downloading it for a delat/full backup or by cloning it for a disaster recovery of continous replication and then wait for the guest tools to be loaded before the end of a timeout of 10 minutes (boot + guest tools).
+
+A VM without guest tools will fail its health check.
+
+The restored VM is then deleted.
+
+#### Execute a script
+
+If a VM has the tag **xo-backup-healthcheck-xenstore** during a backup health check, then XO will wait for a script to change the value of the xenstore `vm-data/xo-backup-health-check` key to be either `success` or `failure`.
+
+In case of `failure`it will mark the health check as failed, and will show the (optionnal) message contained in `vm-data/xo-backup-health-check-error`
```

Comment:
```suggestion
In case of `failure`, it will mark the health check as failed, and will show the (optional) message contained in `vm-data/xo-backup-health-check-error`
```